src/main/java/com/jmb/springfactory/model/dto/ProductionOrderDTO.java

```diff
@@ -13,8 +18,16 @@
 
   @NotNull
   private String name;
+
   @NotNull
   private String description;
+
   @NotNull
   private String measurements;
+  
+  private String status;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "status" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)